brave/browser/brave_browser_context.cc

```diff
@@ -164,7 +164,7 @@ BraveBrowserContext::BraveBrowserContext(
 
   if (in_memory) {
     original_context_ = static_cast<BraveBrowserContext*>(
-        atom::AtomBrowserContext::From(partition, false));
+        atom::AtomBrowserContext::From("", false));
```

Comment:
Sets the `original_context (parent)` of the new in_memory partition to the default partition "". If you use a different partition name you have to specify the parent partition.

Reply:
having more than one incognito context associated with the same original context will cause other issues because `ExtensionsBrowserClient::GetOffTheRecordContext` can only return one browser context for the lazy event dispatcher